calculator-app/src/Buttonnumbers.js

```diff
@@ -0,0 +1,66 @@
+import './App.css';  // import App.css to get the class
+import { useContext  } from "react";   // import useContext 
+import { ButtonContext } from "./App";    // import ButtonContext which was created in the App.js
+
+function Buttonnumbers() {
+  const buttoncont = useContext(ButtonContext);   // Define useContext with imported context (ButtonContext)
+  const {digit, setDigit}=buttoncont      // destructuring the (buttoncont) context variable which are needed (values need to be changed)
+  const numbers = ['/','*','-','+',9,8,7,4,5,6,1,2,3,0,'%','.','='];    // to create array element
+  const operators = ['/','*','-','+','%','.','='];
+  const removeString= (digi) => {
+      if (digi.slice(-1)=== ' ')
```

Comment:
flower bracket should be on write side of block scope.
Please update all over the files